FluentTerminal.App/Views/TerminalView.xaml

```diff
@@ -18,111 +18,149 @@
-                </Button>
+                    </ToggleButton>
 
-                <Button
-                    x:Uid="CloseSearchPanel"
-                    Background="Transparent"
-                    Command="{x:Bind ViewModel.CloseSearchPanelCommand}"
-                    ToolTipService.ToolTip="Close search panel">
-                    <Button.Content>
+                    <ToggleButton
+                        x:Uid="SearchWithRegex"
+                        Background="Transparent"
+                        IsChecked="{x:Bind ViewModel.SearchWithRegex, Mode=TwoWay}"
+                        ToolTipService.ToolTip="Regex">
                         <Viewbox Width="16" Height="16">
-                            <SymbolIcon Symbol="Cancel" />
+                            <TextBlock Text=".*" />
                         </Viewbox>
-                    </Button.Content>
-                </Button>
-            </StackPanel>
-        </controls:DropShadowPanel>
+                    </ToggleButton>
+
+                    <Button
+                        Name="PreviousButton"
+                        x:Uid="FindPrevious"
+                        Margin="6,0,0,0"
+                        Background="Transparent"
+                        Command="{x:Bind ViewModel.FindPreviousCommand}"
+                        ToolTipService.ToolTip="Find previous">
+                        <Button.Content>
+                            <Viewbox Width="16" Height="16">
+                                <FontIcon FontFamily="{StaticResource SymbolThemeFontFamily}" Glyph="&#xE971;" />
+                            </Viewbox>
+                        </Button.Content>
+                    </Button>
+                    <Button
+                        x:Uid="FindNext"
+                        Margin="0,0,6,0"
+                        Background="Transparent"
+                        Command="{x:Bind ViewModel.FindNextCommand}"
+                        ToolTipService.ToolTip="Find next">
+                        <Button.Content>
+                            <Viewbox Width="16" Height="16">
+                                <FontIcon FontFamily="{StaticResource SymbolThemeFontFamily}" Glyph="&#xE972;" />
+                            </Viewbox>
+                        </Button.Content>
+                    </Button>
+
+                    <Button
+                        x:Uid="CloseSearchPanel"
+                        Background="Transparent"
+                        Command="{x:Bind ViewModel.CloseSearchPanelCommand}"
+                        ToolTipService.ToolTip="Close search panel">
+                        <Button.Content>
+                            <Viewbox Width="16" Height="16">
+                                <SymbolIcon Symbol="Cancel" />
+                            </Viewbox>
+                        </Button.Content>
+                    </Button>
+                </StackPanel>
+            </controls:DropShadowPanel>
+            <controls:DropShadowPanel
```

Comment:
@felixse  New `DropShadowPanel` is wrapped into StackPanel along with overlay for Search.